### PR TITLE
Add mypy checks

### PR DESCRIPTION
**Pull request by sbrunner**
Add first type

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -166,6 +166,17 @@ flake8:
 		--copyright-min-file-size=1 \
 		--copyright-regexp="Copyright \(c\) ([0-9][0-9][0-9][0-9]-)?$(shell date +%Y), Camptocamp SA"
 
+.PHONY: mypy
+mypy:
+	MYPYPATH=/usr/local/lib/python3.6/site-packages:/c2cwsgiutils \
```
Comment (gberaudo) [inline Makefile]: Should be 3.5.

Comment (sbrunner) [reply Makefile]: Why?

Comment (sbrunner) [reply Makefile]: I't use on Docker who are based on Python 3.6 ...

Comment (gberaudo) [reply Makefile]: Ah OK, I missed you are only calling the mypy target inside docker.

**Pull request merged**